.github/workflows/neural-sync.yml

```
@@ -3,8 +3,12 @@ name: Neural Branch Sync
 on:
   push:
     branches: ['**']
+    paths-ignore:
+      - 'neural-links/**'
   pull_request:
     types: [opened, closed, merged, synchronize]
```

actionlint (1.7.12)

[error] invalid activity type "merged" for "pull_request" Webhook event. available types are "assigned", "auto_merge_disabled", "auto_merge_enabled", "closed", "converted_to_draft", "demilestoned", "dequeued", "edited", "enqueued", "labeled", "locked", "milestoned", "opened", "ready_for_review", "reopened", "review_request_removed", "review_requested", "synchronize", "unassigned", "unlabeled", "unlocked"

(events)